collector/config.yaml

```diff
@@ -7,6 +7,11 @@ receivers:
 exporters:
   logging:
     loglevel: debug
+  awsprometheusremotewrite:
+    endpoint: "https://aps-workspaces.eu-west-1.amazonaws.com/workspaces/ws-12345678-1234-1234-1234-123456789012/api/v1/remote_write"
+    aws_auth:
+      service: "aps"
+      region: "eu-west-1"
```

Comment:
Is this a real endpoint that is used? If it's not, would this cause a problem if downstream repositories are using this file?

Reply:
I suspect this should be replaced with actual workspace from customer.
I think one addition to the PR would be a readme/documentation about what is added and how to use it

Reply:
This was a sample number to show how to fill the endpoint URL - I will add a comment

Reply:
Thank you @Aneurysm9 - I'll update those